decidim-meetings/app/commands/decidim/meetings/admin/copy_meeting.rb

```diff
@@ -62,7 +62,21 @@ def copy_meeting!
             questionnaire: form.questionnaire,
             registrations_enabled: meeting.registrations_enabled,
             available_slots: meeting.available_slots,
-            registration_terms: meeting.registration_terms
+            registration_terms: meeting.registration_terms,
+            online_meeting_url: meeting.online_meeting_url,
+            type_of_meeting: meeting.type_of_meeting,
+            iframe_embed_type: meeting.iframe_embed_type,
+            iframe_access_level: meeting.iframe_access_level,
+            comments_enabled: meeting.comments_enabled,
+            comments_start_time: meeting.comments_start_time,
+            comments_end_time: meeting.comments_end_time,
+            registration_type: meeting.registration_type,
+            registration_url: meeting.registration_url,
+            attending_organizations: meeting.attending_organizations,
+            reserved_slots: meeting.reserved_slots,
+            customize_registration_email: meeting.customize_registration_email,
+            registration_form_enabled: meeting.registration_form_enabled,
+            registration_email_custom_content: meeting.registration_email_custom_content
```

Comment:
This actually introduces a bug where if you modify any of this fields in the form, then you still have the contents of the original meeting. This actually currently happens too with scope and category (lines 47 and 48), that we need to change it too. 

The tricky part is that we're introducing some fields that aren't in the form, but that are in the model in others forms (like Registration form and `CloseMeetingForm`), so I think the best approach would be to have those fields better explained with its own method. I was thinking something like this: 

```ruby
diff --git a/decidim-meetings/app/commands/decidim/meetings/admin/copy_meeting.rb b/decidim-meetings/app/commands/decidim/meetings/admin/copy_meeting.rb
index 1c2d4b65f7..e909418f1e 100644
--- a/decidim-meetings/app/commands/decidim/meetings/admin/copy_meeting.rb
+++ b/decidim-meetings/app/commands/decidim/meetings/admin/copy_meeting.rb
@@ -44,10 +44,11 @@ module Decidim
           @copied_meeting = Decidim.traceability.create!(
             Meeting,
             form.current_user,
-            scope: meeting.scope,
-            category: meeting.category,
+            component: meeting.component,
             title: parsed_title,
             description: parsed_description,
+            scope: form.scope,
+            category: form.category,
             end_time: form.end_time,
             start_time: form.start_time,
             address: form.address,
@@ -55,28 +56,19 @@ module Decidim
             longitude: form.longitude,
             location: form.location,
             location_hints: form.location_hints,
-            component: meeting.component,
             private_meeting: form.private_meeting,
             transparent: form.transparent,
             author: form.current_organization,
-            questionnaire: form.questionnaire,
-            registrations_enabled: meeting.registrations_enabled,
-            available_slots: meeting.available_slots,
-            registration_terms: meeting.registration_terms,
-            online_meeting_url: meeting.online_meeting_url,
-            type_of_meeting: meeting.type_of_meeting,
-            iframe_embed_type: meeting.iframe_embed_type,
-            iframe_access_level: meeting.iframe_access_level,
-            comments_enabled: meeting.comments_enabled,
-            comments_start_time: meeting.comments_start_time,
-            comments_end_time: meeting.comments_end_time,
-            registration_type: meeting.registration_type,
-            registration_url: meeting.registration_url,
-            attending_organizations: meeting.attending_organizations,
-            reserved_slots: meeting.reserved_slots,
-            customize_registration_email: meeting.customize_registration_email,
-            registration_form_enabled: meeting.registration_form_enabled,
-            registration_email_custom_content: meeting.registration_email_custom_content
+            online_meeting_url: form.online_meeting_url,
+            type_of_meeting: form.type_of_meeting,
+            iframe_embed_type: form.iframe_embed_type,
+            iframe_access_level: form.iframe_access_level,
+            comments_enabled: form.comments_enabled,
+            comments_start_time: form.comments_start_time,
+            comments_end_time: form.comments_end_time,
+            registration_type: form.registration_type,
+            registration_url: form.registration_url,
+            **fields_from_meeting
           )
         end
 
@@ -90,6 +82,17 @@ module Decidim
           end
         end
 
+        def fields_from_meeting
+          {
+            available_slots: meeting.available_slots,
+            registration_terms: meeting.registration_terms,
+            reserved_slots: meeting.reserved_slots,
+            customize_registration_email: meeting.customize_registration_email,
+            registration_form_enabled: meeting.registration_form_enabled,
+            registration_email_custom_content: meeting.registration_email_custom_content
+          }
+        end
+
         def send_notification
           Decidim::EventsManager.publish(
             event: "decidim.events.meetings.meeting_created",
```

A couple of special mentions: 

* questionnaire: As I'm proposing to drop `MeetingCopyForm`, we'll not have questionnaire anymore. We could introduce `copy_questionnaire!` but that'd be out of scope of the current PR. 
* attending_organizations: This is actually used for the CloseMeetingForm, so it doesn't make much sense to copy that when duplicating

Reply:
I see your point... I will have a look ..